rebar.config

```diff
@@ -61,6 +61,12 @@
   {short, [{erl_opts, [{d, 'EQC_SHORT_TEST'}]}]},
   {long, [{erl_opts, [{d, 'EQC_LONG_TEST'}]}]},
   {prod, [{relx, [{dev_mode, false}]}]},
+  {deb, 
+	  [{relx, 
+		  [{dev_mode, false},
+		  {overlay_vars, "rel/vars/deb.config"}]},
+	  {cuttlefish,
```

Comment:
oh no I understand the `deb.config` that makes perfect sense but I wonder why the specific cuttlefish configuration, shouldn't ddb.conf.example always be the file?